no_drama/pip_automation.py

```diff
@@ -30,28 +13,15 @@ def record_req_cached(path):
 
 
 def save_wheels(destination, packages=[], requirements_paths=[]):
-    cache_wheel_command_prefix = "wheel --find-links=wheelhouse --wheel-dir=wheelhouse".split()
     save_wheel_command_prefix = (
-        " wheel --find-links=wheelhouse --no-index --wheel-dir=%s" %
+        " wheel --no-deps --wheel-dir=%s" %
```

Comment:
What does this `--no-deps` do? I see [the docs](https://pip.pypa.io/en/stable/reference/pip_wheel/#cmdoption-no-deps) say that it means "Don't install package dependencies."

In practice, let's say the requirements file includes some package Foo that depends on some other package Bar. Does `--no-deps` mean that only Foo gets built into a wheel, but Bar does not? In that case, how does Bar make it into the build?

Reply:
My thought was that it would mean that every dependency must be spelled out in requirements.txt. I was thinking about the issue from the other day, where eregs was pulling in Django 2.

I'll remove it, though, it was probably a mistake to add that in this PR.

Reply:
It's gone!